### DISCUSSION TITLE
Começando em UX/UI Design

### DISCUSSION THREAD

**Discussion by devigor** [frontendbr/forum#516] [Perguntas e Respostas]
Já faz um tempo que venho acompanhando o crescimento do UX/UI Design, mas só agora resolvi estudar profundamente esse área.
Gostaria de saber o que vocês recomendam para inciar? Tanto blogs, cursos, livros, etc.

**Comment by kvnol** [comment]
Se liga na lista [Awesome Design](https://github.com/gztchan/awesome-design/). 🤘

**Comment by podrivo** [comment]
[Design Para a Internet](http://www.buscape.com.br/design-para-a-internet-felipe-memoria-8535218769) — O livro tem exemplos antigos, mas a percepção ainda é atual.
[A Book Apart](https://abookapart.com/) — Os livros da coleção.
[Designer News](https://www.designernews.co/) — Comunidade de design.
[Movimento UX](http://movimentoux.com/) — Podcast/entrevistas com profissionais da área de UX.

**Comment by camillodev** [comment]
Depende bastante de o quão profundamente você realmente quer estudar.

- [Introdução e Boas Práticas em UX Design](http://fabricio.nu/livro-ux-design/) -  Fabricio Teixeira, bom para quem esta começando mesmo, e quer saber sobre processos. Muitos conteudos desse livro você acha no blog do proprio autor. 
- [Guia completo: como começar em User Experience Design (UX)](https://brasil.uxdesign.cc/guia-completo-como-come%C3%A7ar-em-ux-design-bd21b9f23b8) - O nome ja diz, o melhor guia que conheço. 

Depois dessas duas referencias existem milhares de caminhos a seguir, que depende da sua opnião, da forma que se sente mais a vontade de seguir. 
Existem cursos de pos graduação, e também graduação em Design de Interação atualmente. 
Ou você pode aprender praticando e lendo muitos livres. 
Se quiser seguir carreira como Design de Interação, acho que graduação é pos ajudaria bastante. 
Se é apenas para ter uma boa noção de UX. 
Os livros e artigos citados já daram um otimo norte. 

Alguns livros muito legais também:
- Design of Everythings (Don Norman)
- Don't Make Me Think

Existem mini cursos bem legais também. 
- [UX Design for Mobile Developers](https://br.udacity.com/course/ux-design-for-mobile-developers--ud849/)
- [Introdução a UX aplicada a Apps](https://mva.microsoft.com/pt-br/training-courses/introduo-a-ux-aplicada-a-apps-12110?l=3GHhMVDIB_3804984382)

Mestrado e pós que conheço tem no Senac, IED, Infnet e PUC.
Se for aplicar para algum curso de design, vamos bater um papo, também penso em fazer. 

Boa sorte.

**Comment by felipefialho** [comment]
@zehfernandes @diogorodrigues

**Comment by devigor** [comment]
Opa @camillodev obrigado pelas recomendações!! :smile: 
Gostaria de UX/UI Design para desenhar interfaces de aplicativos, web sites, etc.

**Comment by zehfernandes** [comment]
Ajudei o @dfosco nessa lista no começo do ano. Acho que pode ajudar :)
https://medium.com/vtex-lab/quer-aprender-ux-design-comece-aqui-51c6d7a0c45b

**Comment by felipefialho** [comment]
Complementando @devigor

[UX. E eu com isso?](https://github.com/frontendbr/forum/issues/73) 
[UX e UI ainda não é levado muito a sério?](https://github.com/frontendbr/forum/issues/479) 
[Livros UX/UI](https://github.com/frontendbr/forum/issues/253)  

😉
